SETUP/tests/UserTest.php

```diff
@@ -7,46 +7,14 @@ class UserTest extends PHPUnit\Framework\TestCase
 
     protected function setUp(): void
     {
-        // Attempt to load our test user, if it exists don't create it
-        $sql = "SELECT username FROM users WHERE username = '$this->TEST_USERNAME'";
-        $result = mysqli_query(DPDatabase::get_connection(), $sql);
-        $row = mysqli_fetch_assoc($result);
-        if (!$row) {
-            $sql = "
-                INSERT INTO users
-                SET id = '$this->TEST_USERNAME',
-                    real_name = '$this->TEST_USERNAME',
-                    username = '$this->TEST_USERNAME',
-                    email = '$this->TEST_USERNAME@localhost'
-            ";
-            $result = mysqli_query(DPDatabase::get_connection(), $sql);
-            if (!$result) {
-                throw new Exception("Unable to create test user 1");
-            }
-
-            $sql = "
-                INSERT INTO users
-                SET id = '$this->TEST_USERNAME-2',
-                    real_name = '$this->TEST_USERNAME-2',
-                    username = '$this->TEST_USERNAME-2',
-                    email = '$this->TEST_USERNAME@localhost'
-            ";
-            $result = mysqli_query(DPDatabase::get_connection(), $sql);
-            if (!$result) {
-                throw new Exception("Unable to create test user 2");
-            }
-        } else {
-            mysqli_free_result($result);
-        }
+        create_test_user($this->TEST_USERNAME);
+        create_test_user("$this->TEST_USERNAME-2");
```

Comment:
Silly question: Why is one quoted and the other not? (Since it's that way both here and in the teardown, I assume it's deliberate.)

Reply:
Not a silly question! The "-2" is a string we are appending to the variable so it's in quotes. If it's clearer I can change it to:
```php
create_test_user($this->TEST_USERNAME . "-2");
```

Reply:
I was staring at it after I sent the question, and wondered if that was what it was. I'm fine with it the way that it is -- it just threw me for a bit (remember, I'm still a beginner 😁 ). Thanks!